x-pack/plugins/apm/public/components/shared/ManagedTable/index.tsx

```diff
@@ -39,6 +39,7 @@ interface Props<T> {
     sortDirection: 'asc' | 'desc'
   ) => T[];
   pagination?: boolean;
+  isLoading?: boolean;
```

Comment:
Is this a necessary change? FWICT you're using EuiBasicTable directly but I may have missed something.

Reply:
No, I'm going to rollback it. tks 👍🏻